konfera/models/speaker.py

```diff
@@ -11,7 +10,7 @@ class Speaker(models.Model):
     url = models.URLField()
     social_url = models.URLField()
     country = models.CharField(max_length=64)
-    sponsor = models.ForeignKey(Sponsor, default=None)
+    sponsor = models.ForeignKey('Sponsor', default=None)
 
     def __str__(self):
         return self.title
```

Comment:
This is an error from, that I have missed in review. There is no title in this model, however it might be nice to have optional title field for user with choices (Mr., Ms.). Could you please update function to return title (if set) first_name last_name?

I realize it is not part of your issue, but it would be nice to have it before migration, same goes for ticket model.